docs/examples/yang19.ipynb

```diff
@@ -0,0 +1,573 @@
+    "\n",
+    "In this notebook, we will:\n",
+    "\n",
+    "1. Train an agent on the 20-task suite using supervised learning. We will generate trial data from the environment and train a recurrent neural network to predict the correct choices.\n",
+    "2. Evaluate the agent's performance and behavior across tasks. After training, we will examine how well the agent learned each of the 20 tasks and whether it can flexibly switch contexts from trial to trial.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 0. Install Dependencies\n",
+    "\n",
+    "To begin, install the `neurogym` package. This will automatically install all required dependencies, including Stable-Baselines3.\n",
+    "\n",
+    "For detailed instructions on how to install `neurogym` within a conda environment or in editable mode, refer to the [installation instructions](https://github.com/neurogym/neurogym?tab=readme-ov-file#installation).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Uncomment to install\n",
+    "# ! pip install neurogym[rl]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1. Training an Agent on Yang19 Cognitive Tasks\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.1 Environment Setup and Initial Agent Behavior\n",
+    "\n",
+    "Let's begin by creating and exploring the environment using the `yang19` collection of tasks from NeuroGym. We’ll use the default configuration for all parameters except `dim_ring`, which we set to 2 in order to represent two alternative choices (left/right) arranged on a ring.\n",
+    "\n",
+    "To get a sense of the environment dynamics, we’ll visualize a couple of representative tasks (`dm1` and `multidm`) to better understand their structure and how the agent is expected to interact with them.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1.1 Import Libraries\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "import matplotlib.pyplot as plt\n",
+    "import scipy\n",
+    "import numpy as np\n",
+    "import neurogym as ngym\n",
+    "from neurogym.wrappers import ScheduleEnvs\n",
+    "from neurogym.utils.scheduler import RandomSchedule\n",
+    "from neurogym.utils import plot_env\n",
+    "import gymnasium as gym\n",
+    "import os\n",
+    "import sys\n",
+    "from neurogym.wrappers.monitor import Monitor\n",
+    "import torch.nn as nn\n",
+    "import torch\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")"
```

Comment:
There are a few unused imports here.
Also, I removed the tensorflow warnings, as done for the other notebooks

```suggestion
    "import warnings\n",
    "import matplotlib.pyplot as plt\n",
    "import numpy as np\n",
    "import neurogym as ngym\n",
    "from neurogym.wrappers import ScheduleEnvs\n",
    "from neurogym.utils.scheduler import RandomSchedule\n",
    "from neurogym.utils import plot_env\n",
    "import gymnasium as gym\n",
    "from neurogym.wrappers.monitor import Monitor\n",
    "import torch.nn as nn\n",
    "import torch\n",
    "\n",
    "from IPython.display import clear_output\n",
    "warnings.filterwarnings(\"ignore\")\n",
    "clear_output()"
```